README.md

```diff
@@ -329,7 +331,7 @@ Runtime properties are attached to the tech object when HLS is in
 use. You can get a reference to the HLS source handler like this:
 
 ```javascript
-var hls = player.tech({ IWillNotUseThisInPlugins: true }).hls;
+var hls = player.tech().hls;
```

Comment:
Should we change this and the other `hls` references to `vhs`?

Reply:
We haven't updated all references within the code to have `vhs` yet, e.g. `player.tech().vhs` is `undefined` at the moment